internal-packages/workflow-designer-ui/src/app/app-entry-input-dialog/dialog.tsx

```diff
@@ -206,7 +206,11 @@ export function AppEntryInputDialog({
 	return (
 		<>
 			<div className="flex justify-between items-center mb-[14px]">
-				<div className="flex items-center gap-[12px]"></div>
+				<div className="flex items-center gap-[12px]">
+					<h2 className="text-[20px] font-medium text-text tracking-tight font-sans">
+						All Run
```

Comment:
How about changing this slightly?

```suggestion
						Run App
```